openedx/adg/lms/webinars/models.py

```diff
@@ -392,6 +396,49 @@ def clone(cls, webinar_to_clone):
         webinar_to_clone.co_hosts.set(co_hosts)
         webinar_to_clone.panelists.set(panelists)
 
+    def get_formatted_data(self):
```

Comment:
How about updating method name as this returns data for zoom, not something like which converts object to dict.